runtime/components/jacMachine/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ target_compile_options(${COMPONENT_LIB} INTERFACE
 
 duktape_library(
     TARGET duktape
-    VERSION v2.6.0
+    VERSION fffa346eff06a8764b02c31d4336f63a773a95c3
```

Comment:
Jakou máme motivaci nahradit tag přímo hashem? Dostatečně věřím duktape na to, že nebude čachrovat s tagy a navíc jsou čitelnější.

Reply:
Noo myslím že to s tagem nešlo zbuildit a ten hash jsi mi poradil